markov.py

```diff
@@ -68,7 +69,7 @@ def generate_sentence(self):
             sentence = ""
             for word in new_res:
                 sentence += word + " "
-            sentence += res[-2] + res[-1]
+            sentence += res[-2] + " " + res[-1]
```

Comment:
This adds an unnecessary space before the punctuation of every generated tweet. 
Like this: "Why can't I fix this via ." and "ALL THE THINGS AND WE WILL ."
This should be reverted to exclude this extra space.

Reply:
Fixed. Please test. I'm not sure how Pythonic the fix is, but it seems to work!

Reply:
Not very Pythonic, but it seems to work fine, so it's good by me. Thank you!